test/DurableTask.Netherite.AzureFunctions.Tests/DurableTask.Netherite.AzureFunctions.Tests.csproj

```diff
@@ -24,6 +24,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\DurableTask.Netherite.AzureFunctions\DurableTask.Netherite.AzureFunctions.csproj" />
     <ProjectReference Include="..\DurableTask.Netherite.Tests\DurableTask.Netherite.Tests.csproj" />
+	<PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="3.0.0" />
```

Comment:
nit: can we add this `PackageReference` in a separate/different `ItemGroup` since this `ItemGroup` only has `ProjectReference`s? Also, can we fix the indentation?